driver/registry_default.go

```diff
@@ -368,11 +369,37 @@ func (m *RegistryDefault) CookieManager() sessions.Store {
 		cs := sessions.NewCookieStore(m.c.SecretsSession()...)
 		cs.Options.Secure = !m.c.IsInsecureDevMode()
 		cs.Options.HttpOnly = true
+		if m.c.SessionDomain() != "" {
+			cs.Options.Domain = m.c.SessionDomain()
+		}
+
+		if m.c.SessionPath() != "" {
+			cs.Options.Path = m.c.SessionPath()
+		}
+
+		if m.c.SessionSameSiteMode() != 0 {
+			cs.Options.SameSite = m.c.SessionSameSiteMode()
+		}
+
+		cs.Options.MaxAge = 0
+		if m.c.SessionPersistentCookie() {
+			cs.Options.MaxAge = int(m.c.SessionLifespan().Seconds())
+		}
 		m.sessionsStore = cs
 	}
 	return m.sessionsStore
 }
 
+func (m *RegistryDefault) ContinuityCookieManager() sessions.Store {
+	if m.continuitySessionStore == nil {
+		cs := sessions.NewCookieStore(m.c.SecretsSession()...)
+		cs.Options.Secure = !m.c.IsInsecureDevMode()
+		cs.Options.HttpOnly = true
```

Comment:
Given that `SameSite: None` no longer works with `secure: false` I think we should set the samesite mode of this cookie to `Lax` to avoid any issues (given that it needs to work with cross-origin redirects).